experimental/generation/generator/packages/library/templates/standard/library-global.lu.lg

```diff
@@ -0,0 +1,11 @@
+[generator.lg](../generator.lg)
+
+# filename
+- ${locale}/library/${prefix}-library-global.${locale}.lu
+
+# template
+-```
+> Add examples for operations across properties
+${'#'} ${triggerIntent}
```

Comment:
${'#'} -> #

Reply:
A plain # still messes up the language server error checking so I have been using the escaped version.

---
In reply to: [478258726](https://github.com/microsoft/BotBuilder-Samples/pull/2690#discussion_r478258726) [](ancestors = 478258726)